### PR TITLE
Addition of BusStops enum and listStops command.

### LINKED ISSUE #22: Add list stops command to display all available bus stops
Add the command and its parser.

### PR DESCRIPTION
**Pull request by yuxinwu99**
Fix #22

### CODE REVIEW

File: src/main/java/seedu/duke/BusStops.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    public String getName(){
+        return name;
+    }
+
+    public static void listStops(){
+        for (BusStops info : EnumSet.allOf(BusStops.class)) {
+            System.out.println(info.name + " : " + info.description);
+        }
+    }
+}
```
Comment (yuxinwu99) [inline src/main/java/seedu/duke/BusStops.java]:
```suggestion
package seedu.duke;

import java.util.EnumSet;

public enum BusStops {
    PGP("PGP","prince george residence"),
    KENTRIDGEMRTSTATION("Kent Ridge MRT Station","mrt"),
    OPPUNIVERSITYHEALTHCENTRE("Opp University Health Centre","opposite UHC, where students can get MC"),
    YUSOFISHAKHOUSE("Yusof Ishak House", "Location with student service center and restaurants"),
    CENTRALLIBRARY("Central Library","also known as CLB"),
    KENTRIDGE("Kent Ridge","idk"),
    MUSEUM("Museum", "Opposite Faculty of Engineering"),
    UNIVERSITYTOWN("University Town","place full of fun"),
    UNIVERSITYHEALTHCENTRE("University Health Centre","UHC, where students can get MC"),
    OPPKENTRIDGEMRTSTATION("OppKentRidgeMRTstation","MRT"),
    RAFFLESHALL("Raffles Hall","Hall located nearby Faculty of Engineering"),
    KENTVALE("Kent Vale","somewhere"),
    EA("EA","located in Faculty of Engineering"),
    NUSIT("NUS IT","Opposite Central library"),
    PGPR("PGPR","prince george residence");

    private final String description;
    private final String name;

    BusStops(String name,String description) {
        this.description = description;
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public String getName() {
        return name;
    }

    public static void listStops() {
        for (BusStops info : EnumSet.allOf(BusStops.class)) {
            System.out.println(info.name + " : " + info.description);
        }
    }
}
```

**Review by yuxinwu99 — COMMENTED** [review]
Fix style

File: src/main/java/seedu/duke/Duke.java [hunk 2]
```diff
@@ -5,39 +5,41 @@
 import seedu.duke.ui.Ui;
 
 import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.EnumSet;
```
Comment (wamikamalik) [inline src/main/java/seedu/duke/Duke.java]: Is this package being imported because of the BusStops being set in the array?
Can this be removed?

**Comment by wamikamalik** [comment]
I realized that this will effect other features involving the use of the bus route array in main. We should remember to make appropriate changes in other pull requests that are open

File: src/main/java/seedu/duke/BusStops.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    BusStops(String name,String description) {
+        this.description = description;
+        this.name = name;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public String getName() {
```
Comment (wamikamalik) [inline src/main/java/seedu/duke/BusStops.java:34]: Nice to have this getter here! Ensures that only minor changes will be needed for other features that made use of the route arrays. (hopefully 🤞 )

**Comment by Johnson-Yee** [comment]
Looks good to merge!

File: src/main/java/seedu/duke/Duke.java (hunk 2 above)
Comment (yuxinwu99) [reply src/main/java/seedu/duke/Duke.java]: Yup, only for testing.

Comment (wamikamalik) [reply src/main/java/seedu/duke/Duke.java]:
```suggestion
```

**Review by wamikamalik — APPROVED** [review]
Good to merge!

**Pull request merged**